### DISCUSSION TITLE
Crazy Navigation Issues - Very basic flows breaking with TCA

### DISCUSSION THREAD

**Discussion by SaranjithDev** [pointfreeco/swift-composable-architecture#3486] [General]
### Description

I'm pushing view A from view A itself, and on that I'm pushing view B, now View B actions won't work at all.

every view uses @Presents state

### Checklist

- [X] I have determined whether this bug is also reproducible in a vanilla SwiftUI project.
- [X] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/pointfreeco/swift-composable-architecture/issues) or [discussion](https://github.com/pointfreeco/swift-composable-architecture/discussions).

### Expected behavior

Basic flow should not break

### Actual behavior

Actions are not working in view B

**Comment by mbrandonw** [comment]
Hi @SaranjithDev, there isn't enough information here for anyone to help. Please create a minimal project that reproduces the problem and share it.

Since this isn't an issue with the library I am going to convert it to a discussion. Please feel free to continue the conversation over there!
